frontend/src/components/employer-promotions-block/employer-promotions-block.tsx

```diff
@@ -5,11 +5,12 @@ import SchoolIcon from '@mui/icons-material/School';
 import { routeDynamicSlugFormat } from '@utils/app-url';
 
 interface EmployerPromotionsBlockProps {
+  showBlock?: boolean;
   employerPromotionsBlock?: EmployerPromotionsBlockType;
 }
 
-export default function EmployerPromotionsBlock({ employerPromotionsBlock }: EmployerPromotionsBlockProps) {
-  if (!employerPromotionsBlock?.showBlock) return <></>;
+export default function EmployerPromotionsBlock({ employerPromotionsBlock, showBlock }: EmployerPromotionsBlockProps) {
+  if (!showBlock || !employerPromotionsBlock?.showBlock) return <></>;

```

Comment:
Är prion rätt här?
Jag misstänker att du gjort rätt, men det känns konstigt med not or not vilket bör ge false om man lämnat showblock undefined?

Reply:
Bra fångat, jag justerar